polkadot.json

```diff
@@ -83,37 +83,37 @@
         "imageUrl": "https://i.imgur.com/SHmsZPe.png",
-        "title": "Canary network - explained!",
-        "details": "A canary network is a live network that allows developers to test their blockchain under real-world conditions before launching it on the mainnet. It’s different from a testnet because it carries real value and transactions, providing a more authentic environment for testing. Think of Kusama like a fun and exciting adventure, where developers can take risks and test their limits. This helps create a stronger and more secure Polkadot ecosystem in the end!"
+        "title": "Play on Polkadot",
+        "details": "Polkadot offers a suite of features available to all $DOT holders. You can create and distribution new assets and NFTs, participate in decentralized governance, secure the network and earn rewards via staking, securely bridge to other blockchains, and soon even deploy smart contracts directly to the network."
       },
       {
         "id": 13,
-        "imageUrl": "https://i.imgur.com/mli9QVH.png",
-        "title": "Double the innovation: parallel development on Polkadot and Kusama",
-        "details": "Polkadot and Kusama operate in parallel, with new features and upgrades being tested on Kusama before actually being implemented on Polkadot. That way, developers like you ensure that new features are thoroughly tested and refined before they're released on the main network, providing a better experience for users."
+        "imageUrl": "https://i.imgur.com/ZtKYg13.png",
+        "title": "Parachains",
+        "details": "For everything else, Polkadot has the parachain ecosystem! Parachains ('parallel-chains') are blockchains built by engineering teams in the Polkadot community. They extend the functionality of Polkadot or provide their own custom services. You can find DeFi platforms, prediction markets, NFT marketplaces, and more among the parachain ecosystem."
       },
       {
         "id": 14,
-        "imageUrl": "https://i.imgur.com/ZtKYg13.png",
-        "title": "Their future together",
-        "details": "Together, Polkadot and Kusama form a dynamic duo that encourages fearless exploration and experimentation. Picture your parachains as connections between the island blockchains, constantly growing and evolving, ready to connect with other blockchains and adapt to the ever-changing tides. This allows your parachains to continuously grow and change over time, evolving into something better. The only limit is how fast you can evolve!"
+        "imageUrl": "https://i.imgur.com/YQNFrNk.png",
+        "title": "Kusama - Polkadot's canary network",
+        "details": "Polkadot is arguably the most resilient blockchain system today, and a lot of that is thanks to Kusama. You can think of Kusama as a second public deployment of the Polkadot Network. Developers can deploy features to Kusama before they bring them to Polkadot, ensuring that bugs are found in production before they come to Polkadot."
```

Comment:
```suggestion
        "details": "Polkadot is arguably the most resilient blockchain system today, and much of that can be attributed to Kusama. You can think of Kusama as an early public production-grade deployment of the Polkadot Network. Developers can deploy features to Kusama before they bring them to Polkadot, ensuring that bugs are found in production before they come to Polkadot."
```

Reply:
what does "early" mean here though. I am okay to not say "second", but I think we need to find a better word than both second and early.

Reply:
Rephrased it a bit. May be also highlight the fact that Kusama not only catches bugs in the code, but flaws in the design that can lead to value extraction as KSM has economic value.